package.json

```diff
@@ -21,7 +21,9 @@
     "docs-data": "yarn write-discussion-data Docs",
     "hot-data": "kit update-hot",
     "announcements-data": "yarn write-discussion-data Announcements",
-    "write-repo-scripts-data": "kit write-repo-scripts-data"
+    "write-repo-scripts-data": "kit write-repo-scripts-data",
+    "write-course-data": "kit write-course-data",
+    "course-data": "yarn write-course-data script-kit-showcase-for-optimizing-your-everyday-workflows-e20ceab4"
```

Comment:
script responsible for writing course data. we can add multiple by separating their slugs with comma.